common/lib/capa/capa/util.py

```diff
@@ -195,3 +195,10 @@ def get_inner_html_from_xpath(xpath_node):
     # strips outer tag from html string
     inner_html = re.sub('(?ms)<%s[^>]*>(.*)</%s>' % (xpath_node.tag, xpath_node.tag), '\\1', html)
     return inner_html.strip()
+
+
+def remove_markup(html):
+    """
+    Return the html without markup.
```

Comment:
```
Return html with markup stripped and text HTML-escaped.
```
Feel free to include or not include any of the following additional info in the comment.  It won't double-escape, but it will make sure entities are escaped:
```
>>> bleach.clean("<b>Rock & Roll</b>", tags=[], strip=True)
u'Rock &amp; Roll'
>>> bleach.clean("<b>Rock &amp; Roll</b>", tags=[], strip=True)
u'Rock &amp; Roll'
```